tests/queries/batch_rewards_test_db.sql

```diff
@@ -136,6 +138,13 @@ CREATE TABLE fee_policies (
   PRIMARY KEY (auction_id, order_uid, application_order)
 );
 
+CREATE TABLE app_data (
+  contract_app_data bytea NOT NULL,
+  full_app_data bytea NOT NULL,
+
+  PRIMARY KEY (contract_app_data, full_app_data)
+);
```

Comment:
From the [services repo](https://github.com/cowprotocol/services/blob/8b62faa4ce72836cc17a568a258cda614b75698a/database/sql/V052__add_app_data.sql#L4) it looks as if the table could more concisely defined as follows (the primary key ist just `contract_app_data`):

```suggestion
CREATE TABLE app_data (
    contract_app_data bytea PRIMARY KEY,
    full_app_data bytea NOT NULL
);
```